### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by mjbrisebois**
MVP for working with existing Holochain Conductor.

### CODE REVIEW

**Comment by alastairong** [comment]
CC: @zo-el @SirRobLyon @artbrock

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "module": "build/index.js",
+  "browser": "dist/holo_hosting_web_sdk.js",
+  "scripts": {
+    "build": "npm run compile && npm run bundle",
+    "compile": "tsc --esModuleInterop --lib es2015,dom --outDir ./build -p ./src",
+    "bundle": "npx webpack --mode production --output-filename holo_hosting_web_sdk.js",
+    "test": "npm run test-unit",
+    "test-unit": "mocha ./tests/unit/"
+  },
+  "author": "Holo Inc.",
```
Comment (artbrock) [inline package.json:15]: Holo Ltd.

There is no Holo Inc... FWIW

**Comment by zo-el** [comment]
@artbrock 
I think [this](https://github.com/Holo-Host/web-sdk/pull/1) is the PR with the Development plan

**Comment by alastairong** [comment]
@artbrock The SDK itself will be here too. The intent in getting your input on the readme early is to make you aware that the SDK is more than just a javascript API / library. It also includes the testing environment (hence all the nix stuff) so that UI developers can simply install, code, and test.

One longer term feature idea was to include a Holoscape context too, so that a UI can toggle between different different test environments.

File: package.json (hunk 1 above)
Comment (mjbrisebois) [reply package.json:15]: Thanks, I wasn't sure about that.  I will update it.

**Comment by alastairong** [comment]
@mjbrisebois Can you update who has write access? Joel should at least.

Also can you make Joel a reviewer?

@Joel - reviews are about knowledge transfer as much as code quality. Please review this and #1, and approve it if everything is fine.

CC: @SirRobLyon

**Pull request merged**